### PR TITLE
Transaction error

### PR DESCRIPTION
**Pull request by biswajit-landmarkgroup**

### CODE REVIEW

File: src/Middleware/RecordForegroundTransaction.php [hunk 1]
```diff
@@ -84,7 +85,7 @@ public function terminate($request, $response)
             new RequestProcessedSpan(
                 $this->getTransactionName($request),
                 [
-                    'now' => now()->toDateTimeString(),
+                    'now' => Carbon::now(),
```
Comment (ssi-anik) [inline src/Middleware/RecordForegroundTransaction.php]: Make it `Carbon::now()->toDateTimeString()` please.

**Comment by ssi-anik** [comment]
Please check the review.

File: src/Middleware/RecordForegroundTransaction.php (hunk 1 above)
Comment (biswajit-landmarkgroup) [reply src/Middleware/RecordForegroundTransaction.php]: Done

File: contributor.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Replaced now()->toDateTimeString() with Carbon::now()
+
+I got the below error so changed below lines.
+Uncaught exception 'Error' with message 'Call to undefined function Anik\ElasticApm\Middleware\now()' 
+in /var/www/vendor/anik/elastic-apm-php/src/Middleware/RecordForegroundTransaction.php:87
```
Comment (ssi-anik) [inline contributor.md]: Is this file really necessary?

Comment (biswajit-landmarkgroup) [reply contributor.md]: No. You can remove that. I added for you to know what is the issue.

Comment (biswajit-landmarkgroup) [reply contributor.md]: So you can suggest me the changes needed.

Comment (biswajit-landmarkgroup) [reply contributor.md]: Removed that file

Comment (ssi-anik) [reply contributor.md]: Thanks.

**Comment by ssi-anik** [comment]
Thanks. Tagged `v2.1`
